### PR TITLE
Feat: GET /users/{user_id} api endpoints and tests

### LINKED ISSUE #98: Dev: GET /users/{user_id} api endpoints and tests
## Description
As a student,
I need to create GET /users/{user_id},
so that I can give user the ability to get details of other member.

## Mocks
NA

### Update [Required]
- [ ] user can vieew otheer member's personal details
- [ ] include test cases

### Enhancement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 day

### PR DESCRIPTION
**Pull request by mtreacy002**
### Description
Allow user to get other membre's personal details

Fixes #98 

### Type of Change:
- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
- Make sure you have created 2 users (user a and b) who already verified their email address
- Login as user a
- send GET /userrs/{user_id} request by providing the user b id number
- for successful response (200) you will see below
Screen Shot 2020-07-28 at 11 27 55 pm

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/api/dao/personal_background.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            whether or not user agrees to make their personal background information
+            public to other members of BridgeInTech.
+    
+        Returns:
+                A dictionary containing "message" which indicates whether or not the user_exension 
+                was created successfully and "code" for the HTTP response code.
+        """
+
+        user_id = int(AUTH_COOKIE["user_id"].value)
+        if user_id == 0:
```
Comment (ramitsawhney27) [inline app/api/dao/personal_background.py]: if not user_id

File: app/api/dao/personal_background.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            whether or not user agrees to make their personal background information
+            public to other members of BridgeInTech.
+    
+        Returns:
+                A dictionary containing "message" which indicates whether or not the user_exension 
+                was updated successfully and "code" for the HTTP response code.
+        """
+
+        user_id = int(AUTH_COOKIE["user_id"].value)
+        if user_id == 0:
```
Comment (ramitsawhney27) [inline app/api/dao/personal_background.py]: if not user_id

**Review by ramitsawhney27 — DISMISSED** [review]

File: app/api/dao/personal_background.py (hunk 1 above)
Comment (mtreacy002) [reply app/api/dao/personal_background.py]: done. It's now the EAFP try catch 😉

File: app/api/dao/personal_background.py (hunk 2 above)
Comment (mtreacy002) [reply app/api/dao/personal_background.py]: done. changed to EAFP

**Review by ramitsawhney27 — DISMISSED** [review]

**Review by foongminwong — APPROVED** [review]
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses were tested as below:

   * _Test PR 99 - GET /users/{user_id} api endpoints and tests_  
      _Screenshot/gif/url_:  

     test-bit-backend-pr-99

      _Expected Result_:  As a user, I should be able to get a verified user's information with specified user ID.
      _Actual Result_: Same as expected.

3. OS Version: Windows 10

**Comment by mtreacy002** [comment]
@ramitsawhney27 . Can you please re-approve and merge this PR? Thanks

**Review by ramitsawhney27 — APPROVED** [review]

**Pull request merged**